### PR TITLE
REPOSITORY: Add `Package`

### LINKED ISSUE #32: REPOSITORY: Add `Package`

### PR DESCRIPTION
**Pull request by Ulugbek4499**
closes #32

### CODE REVIEW

File: ConsoleUI/Program.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
-﻿namespace ConsoleUI
+﻿using System;
```
Comment (Xurshid-A-J) [inline ConsoleUI/Program.cs:1]: Bring Header file

**Review by Xurshid-A-J — CHANGES_REQUESTED** [review]

File: CountPad.Domain/Models/Packages/Package.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using CountPad.Domain.Models.Distributors;
+﻿using System;
```
Comment (Xurshid-A-J) [inline CountPad.Domain/Models/Packages/Package.cs:1]: Header file here too

File: CountPad.Domain/Models/Solds/Sold.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿using CountPad.Domain.Models.Users;
+﻿using System;
```
Comment (Xurshid-A-J) [inline CountPad.Domain/Models/Solds/Sold.cs:1]: Header file here too

File: CountPad.Domain/Models/Users/User.cs [hunk 4]
```diff
@@ -1,4 +1,6 @@
-﻿namespace CountPad.Domain.Models.Users
+﻿using System;
```
Comment (Xurshid-A-J) [inline CountPad.Domain/Models/Users/User.cs:1]: Header

File: CountPad.Infrastructure/Repositories/BaseRepository.cs [hunk 5]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
```
Comment (Xurshid-A-J) [inline CountPad.Infrastructure/Repositories/BaseRepository.cs:1]: Header

File: CountPad.Infrastructure/Repositories/PackageRepository.cs [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                    Product = entity.Product.Id,
+                    Count = entity.Count,
+                    Distributor = entity.Distributor.Id,
+                    IncomingPrice = entity.IncomingPrice,
+                    SalePrice = entity.SalePrice,
+                    IncomingDate = entity.IncomingDate
+                });
+            }
+        }
+
```
Comment (Xurshid-A-J) [inline CountPad.Infrastructure/Repositories/PackageRepository.cs:46]: remove extra lines

File: CountPad.Infrastructure/query.txt [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  'books',
+  'furniture',
+  'cosmetics'
+);
+CREATE TYPE user_status AS ENUM (
+    'Saller',
+    'Manager',
+    'Storekeeper'
+);
+
```
Comment (Xurshid-A-J) [inline CountPad.Infrastructure/query.txt:19]: You can remove all enums as we have deleted it already , no need for enum here , they will be as test

**Review by MirzayevDeveloper — APPROVED** [review]
Done!

File: CountPad.Domain/Models/Products/ProductTypes.cs [hunk 8]
```diff
@@ -1,4 +1,9 @@
-﻿using System;
+﻿// --------------------------------------------------------
+// Copyright (c) Coalition of Good-Hearted Engineers
+// Developed by CountPad Team
+// --------------------------------------------------------
+
+using System;
```
Comment (MirzayevDeveloper) [inline CountPad.Domain/Models/Products/ProductTypes.cs:6]: usingdan keyin bitta bush line

File: CountPad.Infrastructure/Repositories/PackageRepository.cs [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using CountPad.Application.Interfaces.RepositoryInterfaces;
+using CountPad.Application.Interfaces.RepositoryInterfaces;
+using CountPad.Domain.Models.Distributors;
+using CountPad.Domain.Models.Packages;
+using CountPad.Domain.Models.Products;
+using Dapper;
+using Npgsql;
+
```
Comment (MirzayevDeveloper) [inline CountPad.Infrastructure/Repositories/PackageRepository.cs:18]: remove extra line

File: CountPad.Infrastructure/Repositories/PackageRepository.cs [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+                            { "SalePrice", entity.SalePrice },
+                            { "IncomingDate", entity.IncomingDate },
+                       });
+
+                return affectedRows;
+            }
+        }
+      /*  id uuid NOT NULL,
+    product_id uuid NOT NULL References products(id),
+    count DOUBLE PRECISION NOT NULL,
```
Comment (MirzayevDeveloper) [inline CountPad.Infrastructure/Repositories/PackageRepository.cs:53]: remove it

Comment (MirzayevDeveloper) [reply CountPad.Infrastructure/Repositories/PackageRepository.cs:53]: do code cleanup

**Pull request merged**